charts/pulsar/templates/bookkeeper-statefulset.yaml

```diff
@@ -133,6 +133,7 @@ spec:
         - >
           bin/apply-config-from-env.py conf/bookkeeper.conf;
           bin/apply-config-from-env.py conf/pulsar_env.sh;
+          cp /logconf/log4j2.yaml {{ template "pulsar.home" .}}/conf;
```

Comment:
Why not just mount the config map directly to the target path?

Reply:
Because the `/sn-platform/conf` folder already exists, if you mount the path here, all the original files will be overwritten.